specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -43,38 +39,11 @@ paths:
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/expand"
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
         - in: query
-          name: kadastralegemeente__code
-          description: "De code van de kadastrale gemeente, deel van de kadastrale aanduiding van de onroerende zaak. De code en waarde zijn vastgelegd in een [Waardelijst](http://www.kadaster.nl/schemas/waardelijsten/KadastraleGemeente/)"
+          name: kadastraleaanduiding
+          description: "Kadastrale aanduiding is een unieke aanduiding van een onroerende zaak. De volgorde waarin deze string wordt opgebouwd is Kadastrale gemeente, perceelnummer, sectie en indien van toepassing appartementsrechtvolgnummer. "
```

Comment:
ik denk dat we de gebruiker iets meer moeten helpen met het juiste invoeren. Worden de componenten direct achter elkaar gezet? Of met een spatie ertussen? Of een ander teken ertussen?

Kunnen we hier geen pattern op definiëren? En een example toevoegen?

Reply:
Ik heb de omschrijving aangepast, een minLength toegevoegd en een Pattern toegevoegd. 
Daarbij ga ik er van uit dat : 
- een kadastralegemeentecode altijd uit 3 hoofdletter en 2 cijfers bestaat (ik kon geen uitzondering vinden in de waardelijst)
- er voorloopnullen voor het perceelnummer en het appartementsrechtvolgnummer worden gezet
- er nullen worden ingevuld als er geen appartmentsrechtvolgnummer is.
- een Kadastraleaanduiding dus altijd 16 lang is. 
- een sectie uit hoofdeletters, kleine letters en cijfers kan bestaan. 

@mhebbink-kad Zou jij deze uitgangspunten kunnen verifieren ?